cell_guide_local_pipeline/cell_guide_data_pipeline.ipynb

```diff
@@ -83,7 +83,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sn = \"prod-snapshot\"\n",
+    "sn = \"staging-snapshot\"\n",
```

Comment:
is the change to staging snapshot intentional here?

Reply:
we can ignore changes to the notebook. ty for pointing this out though!